tools/setup_helpers/cuda.py

```diff
@@ -1,17 +1,35 @@
-import ctypes.util
 import os
+import platform
+import ctypes.util
+from subprocess import Popen, PIPE
 
 from .env import check_env_flag
 
+
+def find_nvcc():
+    proc = Popen(['which', 'nvcc'], stdout=PIPE, stderr=PIPE)
+    out, err = proc.communicate()
+    out = out.decode().strip()
+    if len(out) > 0:
+        return os.path.dirname(out)
```

Comment:
It will, but `ctypes.util.find_library('cudart')` will also give you `$CUDA_HOME/lib`. Later I do `os.path.dirname(cuda_path)`, where `cuda_path` should be either `$CUDA_HOME/bin` or `$CUDA_HOME/lib` so we get `$CUDA_HOME` in either case.